spacemesh/v2/node.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package spacemesh.v2;
+
+message NodeStatusRequest {}
+
+message NodeStatusResponse {
+  uint64 connected_peers = 1; // number of connected neighbors
+  bool   is_synced = 2; // true when meshed is synced
```

Comment:
I think we need one or two additional modes here:

- status unknown
- offline
- syncing
- synced